src/jit/compiler.h

```diff
@@ -1771,7 +1773,11 @@ class Compiler
     // a PSPSym for functions with any EH.
     bool ehNeedsPSPSym() const
     {
+#ifdef _TARGET_X86_
```

Comment:
Why is this change needed? We use the PSPSym for for ARM32, ARM64 and amd64, so why should it be different for x86 Linux?